recipes/oommf/bld.bat

```diff
@@ -0,0 +1,5 @@
+curl -o oommf.zip  "http://math.nist.gov/oommf/dist/oommf12b0_20160930_86_x64.zip"
```

Comment:
Okay, so when oommf12b0_20160930_86_x64.zip is unzipped, it creates a subdirectory `oommf` which contains all files. My understanding is that the whole directory needs to go to `%PREFIX%\opt`.

So the right unix command would be 
- `mv oommf %PREFIX%\opt` or
- `mkdir %PREFIX%\opt/oommf && mv oommf/* %PREFIX%\opt/oommf`

As I haven't got a windows machine for testing, can somebody suggest a suitable command to achieve this in reply to this comment? (@takluyver, @rpep, @mb1a15).